pytest_fixtures/core/broker.py

```diff
@@ -94,6 +93,16 @@ def factory(retry_limit=3, delay=300, workflow=None, **broker_args):
     return factory
 
 
+@pytest.fixture(scope='session')
+def satellite_factory():
+    return _satellite_factory()
+
+
+@pytest.fixture(scope='module')
+def module_satellite_factory():
+    return _satellite_factory()
```

Comment:
Move both of these to `sat_cap_factory` as we are spinning up fresh satellites here.